### DISCUSSION TITLE
Where can I see userIds list on Unleash admin panel?

### DISCUSSION THREAD

**Discussion by baikadamovagauhar** [Unleash/unleash#5975] [Q&A] [answered]
Hi! Can you help me with my question?
I'm integrating unleash with flutter app , and when users are login in to my app I'm sending their userId to Unleash. Where can I see those user ids in Unleash admin panel? And can I remove their id when user logout from app? Or it is not neccessary?

**Comment by sighphyre** [comment, marked as answer]
Hey @baikadamovagauhar, userIds are never sent to Unleash itself, this is part of our privacy guarantees.

You shouldn't need to do anything here - userIds never leave your SDK so there's no cleanup necessary
